### PR TITLE
Fix indentation in  jenkins docs

### PR DESCRIPTION
**Pull request by takirala**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/ksphere/konvoy/partner-solutions/jenkins/index.md [hunk 1]
```diff
@@ -37,69 +43,69 @@ Your Jenkins instance will deploy.
...
 
 1. Create a `values.yaml` file that can be used to install Jenkins from the upstream helm repo.
 
     ```yaml
     master:
-    useSecurity: false
-    installPlugins: ["prometheus:2.0.6","kubernetes:1.18.2"]
-    csrf:
+      useSecurity: false
+      installPlugins: ["prometheus:2.0.6","kubernetes:1.18.2","workflow-job:2.33","workflow-aggregator:2.6","credentials-binding:1.19","git:3.11.0"]
```
Comment (takirala) [inline pages/ksphere/konvoy/partner-solutions/jenkins/index.md:88]: for future reference:

these plugins were added to make sure this is identical to https://github.com/mesosphere/kubeaddons-enterprise/blob/dev/addons/jenkins/1.x/jenkins.yaml#L32-L37

**Comment by takirala** [comment]
@cmastr can you review this ? Do the failing CI checks (most of them seem to be warnings) needs to pass before merging?

**Comment by cmastr** [comment]
> @cmastr Do the failing CI checks (most of them seem to be warnings) needs to pass before merging?

Not necessarily as they are suggestions, but please make sure you consider each one to improve readability.

**Comment by cmastr** [comment]
@takirala let me know when you're done working on this and ready to merge please.

**Comment by takirala** [comment]
@cmastr i am done working on this.

**Comment by cmastr** [comment]
> @cmastr i am done working on this. (ready to merge )

You can add the ready to merge label once we get these squared away.

**Pull request merged**